api/src/main/java/org/apache/cloudstack/api/response/NetworkResponse.java

```diff
@@ -495,4 +503,12 @@ public Date getCreated() {
     public void setCreated(Date created) {
         this.created = created;
     }
+
+    public void setBytesReceived(final Long bytesReceived) {
```

Comment:
I don't fully understand why the key 'final' is being used here. You don't need to use it here.
am I wrong?